docs/stdlib/set.rst

```diff
@@ -503,10 +511,11 @@ Sets
 
     :index: aggregate
 
-    Generalized boolean :eql:op:`and` applied to the set of *values*.
+    Returns a :eql:type:`bool` value with :eql:op:`and` applied
+    to all values.
 
-    The result is ``true`` if all of the *values* are ``true`` or the
-    set of *values* is ``{}``. Return ``false`` otherwise.
+    The result is ``true`` if all of the ``values`` are ``true`` or the
+    set of ``values`` is ``{}``, with ``false`` returned otherwise.
```

Comment:
```suggestion
    The result is ``true`` if all of the *values* are ``true`` or the
    set of *values* is ``{}``, with ``false`` returned otherwise.
```